src/components/Calendar/Toolbar/ExportCalendar.tsx

```diff
@@ -228,13 +227,13 @@ const exportCalendar = () => {
             // Add VEvent to events array
             events.push({
                 productId: 'antalmanac/ics',
-                startOutputType: 'local',
-                endOutputType: 'local',
+                startOutputType: 'local' as const,
+                endOutputType: 'local' as const,
                 title: `${deptCode} ${courseNumber} ${sectionType}`,
                 description: `${courseTitle}\nTaught by ${instructors.join('/')}`,
                 location: `${meeting.bldg}`,
-                start: firstClassStart,
-                end: firstClassEnd,
+                start: firstClassStart as [number,number,number,number,number],
```

Comment:
I wonder if we should abstract this to it's own type alias, so it's more descriptive and we don't have to repeat `[number,number,number,number,number]` all over the place.

Something like: 

```ts
type DateTimeArray = [number,number,number,number,number];
```

Reply:
The type should probably also have a comment explaining that it's `[YEAR, MONTH, DAY, HOUR, MINUTE]`